src/main/webapp/search-forms/editor.tsx

```diff
@@ -90,6 +90,9 @@ const queryToSearch = (query: QueryType) => {
     filterTree,
     srcs: srcs || ['ddf.distribution'],
     sorts: (sorts || []).map(sort => {
+      if (typeof sort !== 'string') {
```

Comment:
What does this do?

Reply:
the sorts that come from basic search are objects that have the form {attribute, direction} already, rather than a string

Reply:
Can we leave a comment for that?